src/shared-components/forms/style.js

```diff
@@ -0,0 +1,99 @@
+import styled from '@emotion/styled';
```

Comment:
there are places in this file that we could use constants like spacing and transition timing